packages/flowtip-core/src/Rect.tsx

```diff
@@ -85,7 +82,10 @@ class Rect {
    * @param   {Object} [b] A rect-like object.
    * @returns {boolean} True if rects are equal.
    */
-  static areEqual(a: ?RectLike, b: ?RectLike): boolean {
+  static areEqual(
+    a: RectLike | null | undefined,
+    b: RectLike | null | undefined,
```

Comment:
is `| null | undefined` preferred to `| void`?

Reply:
Ah nice didn't realize. You can't assign to `void`, but it works for arguments.

```ts
let v: void = null; // bad
```

But:

```ts
const foo = (v: void) => {};
foo(null); // ok
```

Changing where applicable.

Reply:
Wait I lied. VSCode gui didn't update. The latter is also not ok. `null` seems to be required where it is accepted as a value.